specification/domainservices/resource-manager/Microsoft.AAD/preview/2025-06-01/domainservices.json

```diff
@@ -0,0 +1,1243 @@
+            "modelAsString": true
+          }
+        },
+        "syncNtlmPasswords": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "default": "Enabled",
+          "description": "A flag to determine whether or not SyncNtlmPasswords is enabled or disabled.",
+          "x-ms-enum": {
+            "name": "SyncNtlmPasswords",
+            "modelAsString": true
+          }
+        },
+        "syncKerberosPasswords": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "default": "Enabled",
+          "description": "A flag to determine whether or not SyncKerberosPasswords is enabled or disabled.",
+          "x-ms-enum": {
+            "name": "SyncKerberosPasswords",
+            "modelAsString": true
+          }
+        },
+        "syncOnPremPasswords": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "default": "Enabled",
+          "description": "A flag to determine whether or not SyncOnPremPasswords is enabled or disabled.",
+          "x-ms-enum": {
+            "name": "SyncOnPremPasswords",
+            "modelAsString": true
+          }
+        },
+        "kerberosRc4Encryption": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "default": "Enabled",
+          "description": "A flag to determine whether or not KerberosRc4Encryption is enabled or disabled.",
+          "x-ms-enum": {
+            "name": "KerberosRc4Encryption",
+            "modelAsString": true
+          }
+        },
+        "kerberosArmoring": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "default": "Disabled",
+          "description": "A flag to determine whether or not KerberosArmoring is enabled or disabled.",
+          "x-ms-enum": {
+            "name": "KerberosArmoring",
+            "modelAsString": true
+          }
+        },
+        "ldapSigning": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "default": "Disabled",
+          "description": "A flag to determine whether or not LdapSigning is enabled or disabled.",
+          "x-ms-enum": {
+            "name": "LdapSigning",
+            "modelAsString": true
+          }
+        },
+        "channelBinding": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "default": "Disabled",
+          "description": "A flag to determine whether or not ChannelBinding is enabled or disabled.",
+          "x-ms-enum": {
+            "name": "ChannelBinding",
+            "modelAsString": true
+          }
+        },
+        "syncOnPremSamAccountName": {
```

Comment:
>syncOnPremSamAccountName [](http://example.com/codeflow?start=9&length=24)

could you add better description on what does this field represent ? like what does enabled and disabled mean or resort to for the end user.